src/webgpu/api/validation/copyTextureToTexture.spec.ts

```diff
@@ -244,24 +283,35 @@ g.test('multisampled_copy_restrictions')
     );
   });
 
-g.test('uncompressed_texture_format_equality')
+g.test('texture_format_equality')
   .params(
     params()
-      .combine(poptions('srcFormat', kUncompressedTextureFormats))
-      .combine(poptions('dstFormat', kUncompressedTextureFormats))
+      .combine(poptions('srcFormat', kAllTextureFormats))
+      .combine(poptions('dstFormat', kAllTextureFormats))
   )
   .fn(async t => {
     const { srcFormat, dstFormat } = t.params;
 
+    let deviceForTest: GPUDevice | null;
+    if (t.kBCFormats.includes(srcFormat) || t.kBCFormats.includes(dstFormat)) {
```

Comment:
The members in kUncompressedTextureFormats are in "CompressedTextureFormat" type, while the members in kAllTextureFormats are in "GPUTextureFormat" type, so here I can only use my own-defined kBCFormats to use the function "includes".

Reply:
I think you can use `kAllTextureFormatInfo[srcFormat].extension === 'texture-compression-bc'`